examples/basic-google-map/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Google Maps Setup Example
+
+This is an example setup to show a **basic Google Map** with the Google Maps React Hooks library.
+
+## Instructions
+
+To run this project, clone the Google Maps React Hooks repository locally.
+
+At the root of the repository, run:
+
+```shell
+npm run start:sample-map
+```
+
+**NOTE**:
+To see the examples it is needed to add an `.env` file with an API key in the following format:
```

Comment:
Maybe add a link to the Google Maps documentation on [how to create an API key](https://developers.google.com/maps/documentation/embed/get-api-key#:~:text=Go%20to%20the%20Google%20Maps%20Platform%20%3E%20Credentials%20page.&text=On%20the%20Credentials%20page%2C%20click,Click%20Close.).
```suggestion
To see the examples it is needed to add an `.env` file with a [Google Maps API key](https://developers.google.com/maps/documentation/embed/get-api-key#:~:text=Go%20to%20the%20Google%20Maps%20Platform%20%3E%20Credentials%20page.&text=On%20the%20Credentials%20page%2C%20click,Click%20Close.) in the following format:
```

Reply:
Yes perfect